docs/DeveloperGuide.md

```diff
@@ -1855,42 +1855,696 @@ testers are expected to do more _exploratory_ testing.
 
 1. Initial launch
 
-    1. Download the jar file and copy into an empty folder
+    1. Download the jar file and copy it into an empty folder.
 
-    1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be
-       optimum.
+    2. Run `HomeBoss.jar`{.swift}. If you are unsure how to run a `.jar` file, you may refer to this helpful
+       [guide](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Run-JAR-file-example-windows-linux-ubuntu).
 
-1. Saving window preferences
+    3. First register for HomeBoss using the `register`{.swift} command. So, for example, if you want to register an account
+       with the following details:
 
-    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+       > `USERNAME`: Alex123
+       > `PASSWORD`: AlexIsGreat
+       > `CONFIRM_PASSWORD`: AlexIsGreat
+       > `SECRET_QUESTION`: First Pet Name?
+       > `ANSWER`: KoKo
 
-    1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
+       Type `register --user Alex123 --password AlexIsGreat --confirmPass AlexIsGreat --secretQn First Pet Name?
+       --answer Koko`{.swift} into the Command Box and hit enter. More details on the command can be found [here](#register).
 
-1. _{ more test cases …​ }_
+   4. Expected: Should see the HomeBoss Homepage.
 
-### Deleting a customer
+2. Subsequent launches
 
-1. Deleting a customer while all customers are being shown
+   1. Relaunch the application by running `HomeBoss.jar`{.swift}.
 
-    1. Prerequisites: List all customers using the `list` command. Multiple customers in the list.
+   2. Using the `login`{.swift} command, log in into HomeBoss with the same user details entered earlier.
+      Expected: User is able to log in successfully and see the HomeBoss homepage.
```

Comment:
(This comment might be irrelevant depending on how MarkBind renders it)
Perhaps can standardise the sub-points to use 1. 2. etc to make it consistent with the Initial Launch portion

<img width="962" alt="Screenshot 2023-11-09 at 10 27 43 PM" src="https://github.com/AY2324S1-CS2103T-T13-3/tp/assets/122191046/e7fc462b-e1e1-482e-9d21-ca38615465f0">

Reply:
<img width="170" alt="image" src="https://github.com/AY2324S1-CS2103T-T13-3/tp/assets/46633877/b4f6f258-7ecd-45f7-9596-a98d61aa01bc">
Mark Bind renders it this way, would it be better to change it to a different format?

Reply:
Ahh i see, this works!